.github/workflows/osl_sync_main.yml

```diff
@@ -7,10 +7,8 @@ on:
 
 env:
   UPSTREAM_URL: ${{ vars.UPSTREAM_URL }}
-  IGNORED_FILES: "${{ vars.IGNORED_FILES }}"
   SYNC_BRANCH: "main-sync-$(date +'%Y%m%d-%H%M%S')"
   GITHUB_TOKEN: ${{ secrets.KIE1_TOKEN }}
-  # packages in partition0.txt
   PNPM_FILTER: "-F '@kie-tools/kn-plugin-workflow...' -F '@osl/osl-data-index-ephemeral-image...' -F '@osl/osl-data-index-postgresql-image...' -F '@osl/osl-jobs-service-ephemeral-image...' -F '@osl/osl-jobs-service-postgresql-image...' -F '@osl/osl-management-console-image...' -F '@osl/osl-swf-builder-image...' -F '@osl/osl-swf-devmode-image...'"
```

Comment:
Should we put this env var to the GH Action variables?

Reply:
It should be there already, nice catch! Let me review it.